apps/wso2-digital-wallet/src/index.css

```
@@ -11,14 +11,14 @@
 body {
   margin: 0;
   background: #FAFAF9;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: 'Sora', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Stylelint (17.11.1)

[error] Expected no quotes around "Sora" (font-family-name-quotes)

(font-family-name-quotes)